USE_CASES.md

```diff
@@ -166,3 +167,13 @@ echo $response->statusCode();
 echo $response->body();
 print_r($response->headers());
 ```
+
+<a name="sample"></a>
+# Sample Application on Cloud
+
+## Heroku
+Use the button below to instantly setup your own Simple instance for sending email using sendgrid on Heroku.
+
+<a href="https://heroku.com/deploy?template=https://github.com/pangaunn/sendgrid-php-heroku">
```

Comment:
We need this URL to be hosted in this repo. Could you please make the necessary changes?

Reply:
ok will do.

Reply:
@thinkingserious Are you ok if I push codes in another branch? I have read the heroku document it said it must be root folder, but it supports another branch.

Reply:
How about putting the code in `/examples/deploy/heroku-hello-email/`.

If that does not work, then a branch could work.

Reply:
I've created [new branch](https://github.com/pangaunn/sendgrid-php/tree/heroku-hello-sample) in my forked repository. Could you please create new branch here? so I can make a pull request to new branch.

Reply:
I've tried to make it in subfolder like you said. It will mess up your composer.json. We also need to add 2 files in root directory (Procfile and app.json). So I think new branch would be cleaner in this case.

Reply:
Here you go: https://github.com/sendgrid/sendgrid-php/tree/example-heroku-hello-email

Thanks!